tools/trap/ramdumpParser.py

```diff
@@ -442,7 +441,7 @@ def unwind_frame_using_framepointer(self, frame, debug=False):
 			return -1
 
 		# If the dump is given, Read contents of address from the DUMP file
-		# by using read_word API
+		# By using read_word API
```

Comment:
I think this does not need to be capitalized as the comment line continues from the previous line.